sphinx_gallery/gen_rst.py

```diff
@@ -237,10 +236,11 @@ def extract_thumbnail_number(text):
     # check whether the user has specified a specific thumbnail image
     pattr = re.compile("^\s*#\s*sphinx_gallery_thumbnail_number\s*=\s*([0-9]+)\s*$", flags=re.MULTILINE)
     match = pattr.search(text)
-    try:
+
+    if match is None: # by default, use the first figure created
```

Comment:
I you want to use an inline comment, PEP8 requires two spaces before the #.

Personally I am not a big fan of inline comments and in this particular case, I think it would make more sense to put it on a line by itself above `thumbnail_number = 1`